app/task_routes.py

```diff
@@ -0,0 +1,94 @@
+from app import db
+from app.models.task import Task 
+from datetime import datetime
+from dotenv import load_dotenv
+from flask import Blueprint, jsonify, abort, make_response, request
+import requests  
+import os
+
+load_dotenv()
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try: 
+        model_id = int(model_id)
+    except:
+        abort(make_response({
+            "message": f"{cls.__name__} {model_id} is invalid, please search by task_id."}, 400))
+    task = cls.query.get(model_id)
+    if not task:
+        abort(make_response({"message":
+        f"{cls.__name__} {model_id} does not exist."}, 404))
+    return task 
+
+@tasks_bp.route("", methods=["POST"])
+def create_one_task(): 
+    request_body = request.get_json()   
+    if "title" not in request_body or "description" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+    new_task = Task.from_dict(request_body)
+    db.session.add(new_task)
+    db.session.commit()
+    return {"task": new_task.to_dict()}, 201
+
+@tasks_bp.route("", methods=["GET"])
+def get_all_tasks():
+    sort_query = request.args.get("sort")
+    tasks = (
+        Task.query.order_by(Task.title.desc()).all()
+        if sort_query == "desc"
+        else Task.query.order_by(Task.title).all()
+    ) 
```

Comment:
This works! It's a little difficult to read, though. It might be better to turn this into a traditional conditional statement.

```python
    if sort_query == "desc":
        tasks = Task.query.order_by(Task.title.desc()).all()
    else:
        tasks = Task.query.order_by(Task.title).all()
```